examples/browser/getblocksysfee/demo/demo.js

```diff
@@ -0,0 +1,45 @@
+// Index of the block to get
+const index = 0 // <- very first block ever on neo blockchain!
+
+// Get an instance of Neoscan so we can find a working node
+const provider = new Neon.api.neoscan.instance("TestNet");
+
+// Ensure neon-js only talks to RPC endpoint (Neo node) using HTTPS
+Neon.settings.httpsOnly = true;
+
+function InvokeOperation()
+{
+  clearHtml();
+
+  // Get an RPC Endpoint (Neo Node)
+  provider.getRPCEndpoint().then(nodeUrl => {
+    const client = Neon.default.create.rpcClient(nodeUrl);
+    // change 'hash' below to 'index' to use the index version of getblock
```

Comment:
?

Reply:
I'm sorry. What's the question?

Reply:
I dont see `hash` anywhere below. Maybe its swapped around and meant to change `index` to `hash` to use `hash` version?

Reply:
Oh, yeah, right, that was a comment left over from the template I was using. It's an error. I've removed the line. Good eyes as always snowy. Thanks for your help!